src/classes/PS_IntegrationServiceConfig_TEST.cls

```diff
@@ -146,6 +146,25 @@ public with sharing class PS_IntegrationServiceConfig_TEST {
         System.assertEquals(testMerchantId, service.getMerchantIds());
     }
 
+    @IsTest
+    private static void shouldReturnViewCommitmentURL() {
+        String commitmentId = '11fe3d30-2191-4db7-bcd7-d069fa9f2add';
```

Comment:
We can use `RD2_ElevateIntegrationService_TEST.COMMITMENT_ID` instead of specifying the value.